pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/ingress_reconciler.go

```diff
@@ -432,6 +432,11 @@ func createIngress(isvc *v1beta1.InferenceService, useDefault bool, config *v1be
 		})
 		// Include ingressDomain to the domains (both internal and external) derived by KNative
 		hosts = append(hosts, url.Host)
+
+		// Include additional ingressDomain to the domains (both internal and external)
+		if config.AdditionalIngressDomains != nil && len(*config.AdditionalIngressDomains) > 0 {
```

Comment:
Guess we need to do some validation here before append.
We could relay in the k8s validator api, or, does istio validates it?

Reply:
This one should work the same as `ingressConfig.IngressDomain`. We just read the values from the CM and set them in this `config`. I did not find any validation for other fields. Do you see validation for other fields?

Reply:
We will create or update the ingress returned by `createIngress`. If there is validation issue, the creation or update will fail. That's my assumption, but need to verify.

Reply:
would help to verify if the values passed in additionalIngressDomain are valid URLs. This includes their length and presence of special characters